scripts/create-seed-fixture-heartwood

```diff
@@ -27,25 +27,68 @@ mkdir -p $TEST_REPO_PATH
 
 tar -xf $TEST_REPO_ARCHIVE -C $TEST_REPO_PATH
 
-RAD_HOME=$PALM_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad auth
+### ALICE ###
+
+export RAD_HOME=$ALICE_RAD_HOME
+export RAD_SEED=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
 
-GIT_AUTHOR_NAME="Alice Liddell"
-GIT_AUTHOR_EMAIL="alice@radicle.xyz"
-GIT_COMMITTER_NAME="Alice Liddell"
-GIT_COMMITTER_EMAIL="alice@radicle.xyz"
+export GIT_AUTHOR_NAME="Alice Liddell"
+export GIT_AUTHOR_EMAIL="alice@radicle.xyz"
+export GIT_COMMITTER_NAME="Alice Liddell"
+export GIT_COMMITTER_EMAIL="alice@radicle.xyz"
 
-RAD_SEED=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad auth
+eval $(ssh-agent)
+rad auth
+rad auth # first `rad auth` won't add key to ssh-agent
+radicle-node --listen 0.0.0.0:3444 &
 
 cd $ALICE_CHECKOUT
 
 git clone $TEST_REPO_PATH
 cd $TEST_REPO_NAME
+git checkout main
+
+rad init --name "source-browsing" \
+	--description "Git repository for source browsing tests" \
+	--default-branch "main" \
+	--no-confirm
+
 git checkout feature/branch
+git push rad
+
 git checkout orphaned-branch
-git checkout main
+git push rad
+
+### BOB ###
 
-RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad init --name "source-browsing" --description "Git repository for source browsing tests" --default-branch "main" --no-confirm
-PROJECT_ID=$(RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad .)
+export GIT_AUTHOR_NAME="Bob Belcher"
+export GIT_AUTHOR_EMAIL="bob@radicle.xyz"
+export GIT_COMMITTER_NAME="Bob Belcher"
+export GIT_COMMITTER_EMAIL="bob@radicle.xyz"
+export GIT_COMMITTER_DATE="Mon Nov 21 14:00 2022 +0100"
+
+export RAD_HOME=$BOB_RAD_HOME
+export RAD_SEED=fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe
+
+unset SSH_AUTH_SOCK
+unset SSH_AGENT_PID
+eval $(ssh-agent)
+
+rad auth
+radicle-node --connect z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi@0.0.0.0:3444 &
+
+sleep 1
+
+cd $BOB_CHECKOUT
+rad clone rad:zKtT7DmF9H34KkvcKj9PHW19WzjT
+
+killall radicle-node
+
+cd $TEST_REPO_NAME
+echo "Updated readme" > README.md
+git add README.md
+git commit --message "Update readme" --date "$GIT_COMMITTER_DATE"
+git push rad
 
 cd $BASE_PATH
-tar -cjf palm.tar.bz2 --exclude "post-receive" --exclude "pre-receive" -C $ALICE_RAD_HOME .
+tar -cjf palm.tar.bz2 --exclude "post-receive" --exclude "pre-receive" -C $BOB_RAD_HOME .
```

Comment:
Why are we creating the fixture from Bob's workdir and not using a separate seed node?

Reply:
Bob has everything needed. Why sync with a 3rd node?

Reply:
To check that it works.

Reply:
Seed nodes are a critical part of the architecture and previously we had a lot of problems that only showed up when the data went through one, so it is important to test this scenario.

Reply:
sync and clone is already going through the seed nodes (called `radicle-node`), see [here](https://github.com/radicle-dev/radicle-interface/pull/589/files#diff-f06534a87c78edc1de590220fffc6cdd05edf7b5adef0d4b0f0e3c23ba7f4229R43) and [here](https://github.com/radicle-dev/radicle-interface/pull/589/files#diff-f06534a87c78edc1de590220fffc6cdd05edf7b5adef0d4b0f0e3c23ba7f4229R78). 

a more complex integration test doesn't really belong here in `radicle-interface` and lies on the `heartwood` itself.

Reply:
I know that we're testing it elsewhere as well. But I would appreciate if we could do it here as well, because as I said we had problems in the past with data being mangled that only showed up in the interface. Would it be a lot of work to add it?

Reply:
here's the [3rd node](https://github.com/radicle-dev/radicle-interface/compare/21b3cc087c228898a12c01b51bbb5b583e4b0e49..a7b1031e83d20c213314be9c243740663576b1a9)

Reply:
Thank you!